ci/appveyor-with-compiler.cmd

```diff
@@ -0,0 +1,23 @@
+:: Very simple setup:
+:: - if WINDOWS_SDK_VERSION is set then activate the SDK.
+:: - disable the WDK if it's around.
+
+SET COMMAND_TO_RUN=%*
+SET WIN_SDK_ROOT=C:\Program Files\Microsoft SDKs\Windows
+SET WIN_WDK="c:\Program Files (x86)\Windows Kits\10\Include\wdf"
+ECHO SDK: %WINDOWS_SDK_VERSION% ARCH: %PYTHON_ARCH%
+
+IF EXIST %WIN_WDK% (
+    REM See: https://connect.microsoft.com/VisualStudio/feedback/details/1610302/
+    REN %WIN_WDK% 0wdf
+)
+IF "%WINDOWS_SDK_VERSION%"=="" GOTO main
```

Comment:
And here, possibly, assuming we don't need to compile anything to install any of our packages. But I'll leave it in until everything's set up just in case!